### ISSUE TITLE
Memoria

### ISSUE THREAD

**Issue by Tulmot** [Tulmot/Sklearn-Multilabel#84]
Esta semana se van a dedicar bastantes horas a documentar la memoria.

**Comment by alvarag** [comment]
Es mejor que crees una issue por cada "sección" o capítulo que vayas a documentar. Esto es muy genérico

**Comment by Tulmot** [comment]
Es que tampoco se exactamente a que secciones les dedicaré tiempo, sino lo que hago es cuando me ponga con una sección creo la tarea en ese momento.

**Comment by Tulmot** [comment]
Tengo una duda entiendo que para las citas hay @misc @article y @book, para todas la información que encontrado online utilizo @misc para los artículos que me ha pasado Juan Jose he utilizado @article, como no he utilizado ningun libro el @book no uso nunca esa cita.

Entonces es si lo estoy haciendo bien, de todas formas podeis si quereis echarle un vistazo al archivo bibliografia.bib si quereis ver como lo tengo.

**Comment by alvarag** [comment]
Hay muchos tipos de citas, (ej: https://es.wikipedia.org/wiki/BibTeX#Fichero/Archivo_de_informaci%C3%B3n_bibliogr%C3%A1fica_.bib), pero como bien dices puede que no utilices más que dos o tres.

No obstante, sería raro que no tengas que referenciar ningún libro en Conceptos Teóricos, por ejemplo, o en Técnicas y Herramientas.

**Comment by jjrodriguez** [comment]
OK en cuanto a los tipos.

Para la wikipedia, en el menú de la izquierda aparece una opción de "como citar" o "how to cite" en la que tienes la entrada en formato bibtex. Por ejemplo:

@misc{ wiki:xxx,
    author = "Wikipedia contributors",
    title = "Scikit-learn --- Wikipedia{,} The Free Encyclopedia",
    year = "2017",
    url = "\url{https://en.wikipedia.org/w/index.php?title=Scikit-learn&oldid=817245419}",
    note = "[Online; accessed 3-January-2018]"
  }

Para artículos científicos, normalmente se pueden encontrar los datos en formato bibtex.

Por ejemplo, la siguiente entrada se corresponde con un artículo:

@misc{ multilabel,
       author = "researchgate",
       title = "Multi-Label",
       year = "2014",
       url = "https://www.researchgate.net/publication/313818761_Clasificacion_multi-etiquetas_basada_en_reglas_de_asociacion_de_clases",
       note = "[Online]"
}

El título de ese artículo no es "Multi-Label", el autor no es "researchgate". Si buscas "Clasificación multi-etiquetas basada en reglas de asociación de clases" en scholar.google.com te sale el artículo y debajo "importar el BibTex"

**Comment by Tulmot** [comment]
Vale gracias, pues voy a corregir las citas que tengo mal.

**Comment by Tulmot** [comment]
Esta tarea se ha dedicado la mayor parte del tiempo ha poner los sprints al día y ha añadir citas.
